packages/grafana-ui/src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { Suspense } from 'react';
+import { Portal } from '../Portal/Portal';
+import { css, cx } from 'emotion';
+
+const modalBackdrop = css`
```

Comment:
This reminds me I need to document what I was talking about yesterday during front-end weekly:) This should be returned via function.

Reply:
https://github.com/grafana/grafana/pull/19411